usr/share/rear/prep/NFS4SERVER/default/400_prep_nfs_server.sh

```diff
@@ -0,0 +1,13 @@
+# 400_prep_nfs_server.sh
+#
+# prepare stuff for NFS4SERVER
+#
+
+PROGS+=("${PROGS_NFS4SERVER[@]}")
+
+REQUIRED_PROGS+=("${REQUIRED_PROGS_NFS4SERVER[@]}")
+
+MODULES_LOAD+=("${NFS4SERVER_MODULES[@]}")
+
+# Check if at least one trusted client was specified
+(( "${#NFS4SERVER_TRUSTED_CLIENTS[@]}" > 0 )) || Error "You must have defined at least one client in NFS4SERVER_TRUSTED_CLIENTS."
```

Comment:
You could omit this check from this file and add a symlink pointing to usr/share/rear/verify/NFS4SERVER/default/400_verify_nfs_server.sh to this directory instead to reduce code duplication a bit.

Reply:
I think better to not mess with the `MODULES_LOAD` variable during `verify` because there no code cares about it and I don't like code that is meaningless but appears to be important. It might confuse other people who come to work on ReaR.

Reply:
@schlomo oh, I meant only the last line, not the stuff above.